x/crosschain/keeper/evm_hooks.go

```diff
@@ -293,7 +310,7 @@ func ParseZetaSentEvent(log ethtypes.Log, connectorZEVM ethcommon.Address) (*con
 	}
 
 	if event.Raw.Address != connectorZEVM {
-		return nil, fmt.Errorf("ParseZetaSentEvent: event address %s does not match connectorZEVM %s", event.Raw.Address.Hex(), connectorZEVM.Hex())
+		return event, fmt.Errorf("ParseZetaSentEvent: event address %s does not match connectorZEVM %s", event.Raw.Address.Hex(), connectorZEVM.Hex())
```

Comment:
I think it's okay to return `nil` if you don't want to print too many logs of `Error processing Zeta Sent event , from Address ...` from the caller's side. It's an irreverent `ZetaSent` event emitted from non-connector contract.

Reply:
aaea061766d4b155b32783fcb417ae9388328e95